map/create-and-save-map/src/main/java/com/esri/samples/create_and_save_map/CreateAndSaveMapController.java

```diff
@@ -78,7 +69,6 @@ private void initialize() {
 
     // authentication with an API key or named user is required to access basemaps and other location services
```

Comment:
I just wonder if this comment is a bit confusing here as this line is technically just defining a string. The actual authentication is now happening below